src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,137 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private int arraySize = 16;
```

Comment:
let's make a constant from this value that will be called `DEFAULT_CAPACITY`, you don't need a variable that stores the current length of the array below, you can get current length of array using array itself